docs/source/templates/recipes.rst_t

```diff
@@ -15,19 +15,19 @@ Available packages
     <table id="recipestable" class="display" cellspacing="0" width="100%">
     <thead>
         <tr>
-        {% for key in keys %}
+        {%- for key in keys -%}
         <th>{{ key }}</th>
-        {% endfor %}
+        {%- endfor -%}
         </tr>
     </thead>
     <tbody>
-    {% for r in recipes %}
+    {%- for r in recipes -%}
     <tr>
-        {% for k in keys %}
+        {%- for k in keys -%}
         <td>{{ r[k] }}</td>
-        {% endfor %}
+        {%- endfor -%}
     </tr>
-    {% endfor %}
+    {% endfor -%}
```

Comment:
`{%-`

Reply:
That's intentional, so that each row has its own line. Otherwise, the entire table becomes one line, making it rather hard to look at the HTML should one ever need to.

Reply:
(For reference: the dash adjacent to the percent sign in jinja loop statements makes jinja drop white space in that direction)